socorro/unittest/processor/test_processor_2015.py

```diff
@@ -63,64 +39,30 @@ def test_rule_sets_from_string_1(self):
         assert rc.ruleset01.rule_system_class.default == expected
         assert 'apply_all_rules' == rc.ruleset01.action.default
         expected = (
-            'socorro.processor.support_classifiers.BitguardClassifier, '
-            'socorro.processor.support_classifiers.OutOfDateClassifier'
+            'socorro.processor.general_transform_rules.CPUInfoRule, '
+            'socorro.processor.general_transform_rules.OSInfoRule '
         )
         assert rc.ruleset01.rules_list.default == expected
 
-    def test_rule_sets_from_string_2(self):
```

Comment:
I didn't replace this. It seemed pretty low value, and the follow on patches are likely to move back from rule sets to a rule list, which will necessitate redoing this set of processor_2015 stuff shortly

Reply:
👍